client/src/stylesheets/components/_SignCampaign.scss

```diff
@@ -41,3 +47,45 @@ h2 {
   box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.15) inset, 0 0 6px rgba(0, 0, 0, 0.2) inset;
   border-color: #000\9;
 }
+
+.signature-list-show {
+}
+
+.signature-list-down, .signature-list-up {
+  position: relative;
+  height: 1em;
+  display: inline-block;
+  margin-bottom:.5em;
+}
+
+.signature-list-up:after {
+  content:'';
+  position: absolute;
+  top: 100%;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  width: 0;
+  height: 0;
+  border-top: solid 5px black;
+  border-left: solid 25px transparent;
+  border-right: solid 25px transparent;
+}
+
+.signature-list-hide {
+  display:none;
+}
+
+.signature-list-down:after {
+  content:'';
+  position: absolute;
+  top: 100%;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  width: 0;
+  height: 0;
+  border-bottom: solid 5px black;
```

Comment:
Do we want these to be responsive values or will it not matter once we have breakpoints? 🤔

Reply:
I think these are temporary values until we can do something better, so yes... soon.